webapp/graphite/render/evaluator.py

```diff
@@ -78,6 +78,27 @@ def evaluateTokens(requestContext, tokens, replacements=None):
     raise ValueError("unknown token in target evaluator")
 
 
+def extractPathExpressions(targets):
```

Comment:
added that: https://github.com/graphite-project/graphite-web/pull/1818/commits/14f86796be055eea15790220bcf9d1bb4a6fb9af